README.md

```diff
@@ -36,22 +37,38 @@ See the generated [dbt docs site](https://brooklyn-data.github.io/dbt_artifacts/
 ## Quickstart
 
 1. Add this package to your `packages.yml`:
+
 ```
 packages:
   - package: brooklyn-data/dbt_artifacts
     version: 2.3.0
 ```
 
+:contruction-worker: Make sure to fix at least the **minor** version, to avoid issues when a new release is open. See the notes on upgrading below for more detail.
```

Comment:
```suggestion
:contruction_worker: Make sure to fix at least the **minor** version, to avoid issues when a new release is open. See the notes on upgrading below for more detail.
```